docs/source/install/all_in_one.rst

```diff
@@ -0,0 +1,136 @@
+All-in-one Installation
+=======================
+|st2| provides an all-in-one installer aimed at assisting users with the initial setup and configuration. The installer comes pre-bundled in a number of different provisioning options for convenience, or can also be manually deployed and installed manually on a server.
+
+Pre-Requsites
+-------------
+Before getting started, it is necessary to do a bit of pre-planning to integrate |st2| into an environment. Below is a checklist of items to prepare before running the installer.
+
+#. A server. See below for the list of pre-bundled options, or bring your own.
+#. Hostname of new server setup with DNS.
+#. Admin Password.
+#. Administrative SSH Public/Private Keys *(Optional)*
+#. SSL Certificates *(Optional)*
+#. Chat service connection credentials *(Optional)*
+
+The initial setup is designed to configure itself to a freshly deployed server with no additional services running. If you do not have or do not wish to provide SSL and SSH keys during the initial setup, |st2| will generate random new keys automatically.
+
+Sizing the Server
+-----------------
+A standard all-in-one installation has all of the various components of |st2|, as well as the supporting infrastructure components. While the system can operate with less equipped servers, these are recommended for the best experience while testing or deploying |st2|.
+
+Testing
+~~~~~~~
+* Single or Dual CPU system
+* At least 4GB of RAM
```

Comment:
As I understand now, the idea is: 
* `2GB` is ok enough in general
* but let's keep min `4GB` for this big prod installer

----

Related thing: I want to check (already checking) minimum amount of memory for Ansible provisioner. 
What recommended numbers should I set to keep things in sync? [ [1G](https://github.com/StackStorm/st2workroom/blob/master/stacks/st2.yaml#L5) ], [ [2G](https://github.com/StackStorm/st2workroom/blob/master/stacks/template.yaml#L5) ] or 4 ?